src/main/java/Ui.java

```diff
@@ -0,0 +1,51 @@
+public class Ui {
+    /**
+     * Returns formatted output string for bot message.
+     *
+     * @param input An input string.
+     * @return a formatted string for bot message.
+     */
+    private static String format(String input) {
+        return String.format("\t--------------------- " +
+                "\n\t%1$s--------------------- ", input);
+    }
+
+    /**
+     * Prints greeting message for bot.
+     */
+    public static void greeting() {
```

Comment:
I personally think `greet` would be better suited for this method name! 👍 

```suggestion
    public static void greet() {
```